client/src/ui/modules/military/battle-view/BattleActions.tsx

```
@@ -225,21 +226,33 @@ export const BattleActions = ({
     const maxResourceAmountStolen = getMaxResourceAmountStolen(selectedArmy, defenderArmy, troopConfig);
     const [attackerTroopsLoss, defenseTroopsLoss] = getTroopLossOnRaid(selectedArmy, defenderArmy, troopConfig);
     let content = [
-      <div>Raid outcome:</div>,
-      <div>Your troops loss: {Number(attackerTroopsLoss)}</div>,
-      <div>Defender troops loss: {Number(defenseTroopsLoss)}</div>,
-      <div>Success chance: {raidSuccessPercentage.toFixed(2)}%</div>,
-      <div>Max weight of random resources stolen: {maxResourceAmountStolen}</div>,
+      <div className="text-xs font-bold text-center">Raid outcome:</div>,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)